data/amd/register/control_register/efer.yml

```diff
@@ -0,0 +1,187 @@
+- name: efer
```

Comment:
EFER is not a control register, so this whole file shouldn't be in the `control_register` directory. It is a model specific register (MSR) that behaves like a control register.

See page 55 of: https://www.amd.com/system/files/TechDocs/24593.pdf

```
The EFER is a model-specific register (MSR)
with an address of C000_0080h (see Section 3.2 “Model-Specific Registers (MSRs),” 
```

Reply:
ahh alright, i wasn't sure where to put it since it was in the control register section in the manual. thanks for pointing that out